src/org/jitsi/impl/neomedia/transform/csrc/SsrcTransformEngine.java

```diff
@@ -261,6 +267,17 @@ public RawPacket reverseTransform(RawPacket pkt)
                 csrcAudioLevelDispatcher.addLevels(levels, pkt.getTimestamp());
             }
         }
+
+        SequenceNumberRewriter rewriter = ssrcToRewriter.get(pkt.getSSRCAsLong());
+        if (rewriter == null)
+        {
+            rewriter = new SequenceNumberRewriter();
+            ssrcToRewriter.put(pkt.getSSRCAsLong(), rewriter);
+        }
+
+        rewriter.rewrite(
+            !dropPkt, pkt.getBuffer(), pkt.getOffset(), pkt.getLength());
```

Comment:
This transformer is placed in the chain before SRTP. If it rewrites sequence numbers SRTP authentication will fail :(